.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: scala
-scala:
-- 2.11.6
```

Comment:
Sorry, can you elaborate a little bit, please? I think I followed the docs: http://docs.travis-ci.com/user/languages/scala/